sample-new-architecture/src/App.tsx

```diff
@@ -85,6 +85,9 @@ Sentry.init({
   // otherwise they will not work.
   // release: 'myapp@1.2.3+1',
   // dist: `1`,
+  _experiments: {
+    profileSampleRate: 1,
```

Comment:
Just to be consistent with tracing and browser/node profiling - we pluralize these to `tracesSampleRate` and `profilesSampleRate`

Reply:
Thanks, I've overlooked to Java and there it is in singular.